kodi_addon_checker/check_dependencies.py

```diff
@@ -27,6 +27,17 @@
                       'kodi.resource.font', 'kodi.inputstream', 'kodi.vfs', 'kodi.imagedecoder', 'xbmc.addon',
                       'xbmc.gui', 'xbmc.json', 'xbmc.metadata', 'xbmc.python', 'script.module.pil']
 
+extensions = {"kodi.gameclient": "kodi.binary.instance.game", "kodi.vfs": "kodi.binary.instance.vfs",
+              "xbmc.gui.skin": "xbmc.gui", "xbmc.metadata.scraper.albums": "xbmc.metadata",
+              "xbmc.metadata.scraper.artists": "xbmc.metadata", "xbmc.metadata.scraper.library": "xbmc.metadata",
+              "xbmc.metadata.scraper.movies": "xbmc.metadata", "xbmc.metadata.scraper.musicvideos": "xbmc.metadata",
+              "xbmc.metadata.scraper.tvshows": "xbmc.metadata", "xbmc.pvrclient": "kodi.binary.instance.pvr",
+              "xbmc.python.library": "xbmc.python", "xbmc.python.lyrics": "xbmc.python",
+              "xbmc.python.module": "xbmc.python", "xbmc.python.pluginsource": "xbmc.python",
+              "xbmc.python.script": "xbmc.python", "xbmc.python.weather": "xbmc.python",
+              "xbmc.ui.screensaver": "xbmc.python", "xbmc.webinterface": "xbmc.webinterface",
```

Comment:
`xbmc.ui.screensaver` doesn't always require `xbmc.python`.

A screensaver can either be a binary add-on or a python add-on.

Reply:
So should I remove it from the list?

Reply:
If you do not plan to implement it correct, remove it.